docs/en/advanced/Configuration-Settings.md

```diff
@@ -66,55 +73,26 @@ $ bin/hadoop jar libexec/share/hadoop/mapreduce/hadoop-mapreduce-examples-2.7.3.
 <INPUT FILES> <OUTPUT DIRECTORY>
 ```
 
+### Hive Jobs
```

Comment:
do you think it makes sense to put the compute-specific settings in the compute pages, and tell users to see the doc for their specific compute?

Reply:
I think what we will try to do is add all the complete information in the specific compute pages, and link to them from here.